docs/Deploying/Configuration.md

```diff
@@ -2249,99 +2249,18 @@ Fleet supports the SAML Web Browser SSO Profile using the HTTP Redirect Binding.
 
 _**Note: The email used in the SAML Assertion must match a user that already exists in Fleet unless you enable [JIT provisioning](#just-in-time-jit-user-provisioning).**_
 
-##### sso_settings.enable_sso
```

Comment:
The settings' documentation was moved to the Organization Settings section in the `configuration files` page, for consistency with the rest of the appconfig settings. Backlinks were added here and in the corresponding section in the `configuration files` page.

Reply:
"Backlinks" may not be the right term, maybe more "crosslinks" - what I mean is that the SSO Settings section links to the "Configuring SSO" section on the "Deploying" page:

![sso-settings-config](https://user-images.githubusercontent.com/692384/188639624-75667686-53c2-4145-8806-9cc027669f9c.png)
---

and from there it also links to the "SSO Settings" section on this configuration files page:

![sso-settings-from-deploying-doc](https://user-images.githubusercontent.com/692384/188639622-41448d07-80af-40dd-8c4c-84aa4ffbdf0c.png)

I think that's useful as it links to complementary information found elsewhere in our docs.

Reply:
> "Backlinks" may not be the right term, maybe more "crosslinks"

Ah, ok. When I said "backlinks" I was referencing the new `<span>` elements. I will continue the conversation in the appropriate thread: https://github.com/fleetdm/fleet/pull/7449/files/7a02aa0fceed5db0a1071da0b54e4370d1dd9f2e#r961922107

> I think that's useful as it links to complementary information

I agree. This link you added (from deploying to configuration) is useful:

![Screen Shot 2022-09-06 at 2 17 25 PM](https://user-images.githubusercontent.com/47070608/188709371-30f1a51b-1ff4-4c5b-8efc-bde1e43abb3f.png)